docs/estimators/alchemlyb.estimators.TI_GQ.rst

```diff
@@ -0,0 +1,12 @@
+.. _estimatators_TI_GQ:
+
+TI_GQ
+=====
+The :class:`~alchemlyb.estimators.TI_GQ` estimator is a simple implementation of `thermodynamic integration <https://en.wikipedia.org/wiki/Thermodynamic_integration>`_ that uses the `gaussian quadrature <https://en.wikipedia.org/wiki/Gaussian_quadrature>`_ for integrating the space between :math:`\left<\frac{dH}{d\lambda}\right>` values for each :math:`\lambda` sampled.
+To use this method, make sure that the simulations are performed at certain :math:`\lambda` values using fixed gaussian quadrature points (e.g., [He2020]_). Currently, upto 16 guassian quadrature points are supported. Otherwise, there would be associated errors.
```

Comment:
up to

Reply:
No need for `Otherwise, there would be associated errors.`

Reply:
Might be good to add a rst table here to show the exact lambdas so people can check the doc and know the lambda that are needed.